docs/rst/manual/data_types/index.rst

```diff
@@ -0,0 +1,625 @@
+The data types available in ERT
+===============================
+
+Very briefly described the purpose of ERT is to pass uncertain paramater values
+to a simulator [#]_, in a form which works as suitable input to the simulator and
+then subsequently load the results from the simulator. This means that data must
+be formatted in a form which the simulator can load, and also that ERT must be
+able to read files generated by the simulator.
+
+The data managed by ERT are organized in different *data types* described in
+this chapter. Configuring the data used in the conditioning project is a very
+important part of setting up a ERT configuration file - in practical terms this
+is how you configure which uncertainty parameters should be studied. The data
+types in ert can be categorized in two ways:
+
+1. How the data type behaves dynamically: is it a static parameter like porosity
+   or a relperm parameter - i.e. does it serve as *input* to the simulator, or
+   is it a quantity which is generated as a result from the simulation. When
+   understanding the model updating algorithm and process it is important to
+   understand this difference properly.
+
+2. How the data type is implemented, what type of files does it and read write,
+   how is it configured and so on.
+
+NB: All datatypes have a common namespace; i.e. a globally unique keyword.
+
+
+.. [#] *Simulator* should in this context be understood as the complete
+       forward model, including various pre and post processing steps in
+       addition to the actual reservoir simulation.
+
+
+Parameters
+----------
+
+The *parameter* datatypes will serve as input to the reservoir simulator. For a
```

Comment:
or just the simulator? I must honestly admit, even though I do agree that it is correct, that the usage of _simulator_ instead of _forward model_  was a little bit confusing..